### PR TITLE
Fix typos in PHP cheatsheet

### PR DESCRIPTION
**Pull request by nino**
Found some typos in the PHP cheatsheet.

### CODE REVIEW

File: cheatsheets/gleam-for-php-users.md [hunk 1]
```diff
@@ -765,9 +765,9 @@ between floats and integers in various ways including `rounding`, `floor`,
 
 ### Case
 
-Case is one of the most used control flow in Gleam. It can be seen as a switch
-statement on steroids. It provides a terse way to match a value type to an
-expression. It is also used to replace `if`/`else` statements.
+Case is one of the most used control flow methods in Gleam. It can be seen as a
```
Comment (nino) [inline cheatsheets/gleam-for-php-users.md:768]: Not sure if "methods" is the best word here. I could change it to something like "construct" or "pattern" etc

**Review by lpil — COMMENTED** [review]
Thank you!! I've left one comment

File: cheatsheets/gleam-for-php-users.md (hunk 1 above)
Comment (nino) [reply cheatsheets/gleam-for-php-users.md]: Done!

**Review by lpil — APPROVED** [review]
Thank you!

**Pull request merged**